src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGetServices.cs

```diff
@@ -8,14 +8,14 @@ namespace NuGet.VisualStudio.Internal.Contracts
 {
     public static class NuGetServices
     {
-        public static string NuGetSolutionServiceName = "NuGetSolutionService";
-        public static string NuGetSolutionServiceVersion = "1.0.0";
+        internal static string NuGetSolutionServiceName = "NuGetSolutionService";
```

Comment:
Add an unit test to compare these constants with their equivalents in BrokeredServicesUtility .

Reply:
More of an integration test I guess. 
I can do that. In the interest of time, I might do that in a future PR, but I'll see. 
The builds seem to be failing for some reason.